utils/UniswapPoolsHelpers.ts

```diff
@@ -37,8 +39,9 @@ export async function updateOracle(hre: HardhatRuntimeEnvironment, symbol0: stri
 
   try {
     console.log(`Starting to update ${oracleName}`);
-    await (await oracle.connect(updater).updateOracle()).wait();
-    console.log(`Updated ${oracleName}`);
+
+    // Do not await on the function so we could run multiple transactions like that in parallel
```

Comment:
is it clearly seen when an update transaction fails? since we don't `await` on line 44 the `catch` won't catch it... if we can miss that I think it will be better to wait here to make sure we don't miss anything important

Reply:
I can also add a `.catch` to the promise to print the error.. That will be better as it will still remained all in parallel that way

Reply:
Take a look at it now